geometry_msgs/Point current_position
float64 range
---
geometry_msgs/Vector3[] position
geometry_msgs/Vector3[] velocity
geometry_msgs/Vector3[] size